Makefile

```diff
@@ -0,0 +1,24 @@
+.DEFAULT_GOAL:=help
+
+.PHONY: test
+test: ## Run the integration tests.
+	@./run_test.sh $(target) $(models) $(seeds)
+
+.PHONY: dev
+dev: ## Installs dbt-* packages in develop mode along with development dependencies.
+	@\
+    echo "Install dbt-$(target)..."; \
+    pip install --upgrade pip setuptools; \
+	pip install --pre "dbt-$(target)" -r dev-requirements.txt;
```

Comment:
Do we want this to always be `--pre`? This will include pre-release versions of the dbt adapters (like release candidates).

Or would we want this to typically not include `--pre` by default, but allow it to be configurable?